### PR TITLE
Feature/841

### PR DESCRIPTION
**Pull request by talalus**
#841 

Still need to know what is the best place to track
-  product impressions
-  checkout step

### CODE REVIEW

**Comment by talalus** [comment]
@patrykpiston changelong updated

**Comment by Flyingmana** [comment]
@talalus for changelog files, please dont replace but only add a new line, then git can do its magic to avoid conflicts

File: src/modules/google-tag-manager/hooks/afterRegistration.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import { currentStoreView } from '@vue-storefront/core/lib/multistore'
+
+export function afterRegistration(Vue, config, store, isServer) {
+  if (config.analytics.id && !isServer) {
```
Comment (Flyingmana) [inline src/modules/google-tag-manager/hooks/afterRegistration.ts]: this should check for `config.googleTagManager.id`

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.10.0-rc.1] - UNRELEASED
 
 ### Added
--
```
Comment (Flyingmana) [inline CHANGELOG.md]: replacing this line causes a conflict, it should just add a new one

File: src/modules/google-tag-manager/hooks/afterRegistration.ts (hunk 1 above)
Comment (talalus) [reply src/modules/google-tag-manager/hooks/afterRegistration.ts]: Good catch, Thanks

File: CHANGELOG.md (hunk 2 above)
Comment (talalus) [reply CHANGELOG.md]: done

**Pull request merged**